website/.gitignore

```diff
@@ -1,32 +0,0 @@
-# dependencies
```

Comment:
I'm still seeing `website/*` in:
`.eslintignore`
`.gitignore`
`.prettierignore`

Reply:
I left those in there in case any feature branches accidentally pushed them up, but I guess that shouldn't happen since they'll be pulling from main first anyway